packages/fast-components-styles-msft/src/lightweight-button/index.ts

```diff
@@ -48,21 +48,25 @@ const styles: ComponentStyles<LightweightButtonClassNameContract, DesignSystem>
             background: accentForegroundActive,
         },
         "&$button__disabled, &$button__disabled $button_contentRegion::before": {
-            backgroundColor: "transparent",
+            "background-color": "transparent",
             ...highContrastDisabledForeground,
         },
+        "@media (-ms-high-contrast:active)": {
```

Comment:
Is this intentional, or a merge issue with the HC work?

Reply:
Probably merge issue - good eye. Thanks